test/emf_ecs/ecs_resources/ec2_launch/daemon/ecs_taskdef.tpl

```diff
@@ -0,0 +1,80 @@
+[
```

Comment:
These probably could all be nested as `emf` dir under the existing `ecs` dir instead of creating a new root level dir..
But I think a lot of the structure in this repo needs to change by a lot anyway - so not going to block on it for now.